site/web/assets/js/search.js

```diff
@@ -0,0 +1,171 @@
+function algoliaAutocomplete() {
+  var client = algoliasearch('ENTER_APPLICATION_ID', 'ENTER_APPLICATION_KEY')
```

Comment:
Possible approach:
Transclude Algolia settings into `window.__algoliaConfig__` in a twig template and utilize it by `window.__algoliaConfig__.applicationId`

```html
<script>
window.__algoliaConfig__ = {
  applicationId: "{{ algolia_app_id }}"
  // ... more settings
}
</script>
```